app/api/collection.ts

```diff
@@ -96,20 +86,13 @@ class CollectionAPI extends PlutoAxios {
         throw new Error("Collection API's getPapers method is broken.");
       }
     });
-    const camelizedPaperData = camelcaseKeys(resPaperData, { deep: true });
-    const camelizedPageData = camelcaseKeys(res.data.data.page, { deep: true });
-    const normalizedData = normalize(camelizedPaperData, [paperInCollectionSchema]);
+    const camelizedData = camelcaseKeys(resData.data, { deep: true });
+    const normalizedData = normalize(camelizedData.content, [paperInCollectionSchema]);
     return {
       entities: normalizedData.entities,
       result: normalizedData.result,
-      number: camelizedPageData.page,
-      sort: params.sort,
-      size: camelizedPageData.size,
-      first: camelizedPageData.first,
-      last: camelizedPageData.last,
-      numberOfElements: camelizedPageData.numberOfElements,
-      totalPages: camelizedPageData.totalPages,
-      totalElements: camelizedPageData.totalElements,
+      page: camelizedData.page,
```

Comment:
please sync page number at here.
in Scinapse, we always use page number that starts with 1. (ex: 1, 2, 3, 4, 5,...)
`pageIndex` or `index` should start with 0. (ex: 0, 1, 2, 3, ...)

The only exception is API module.
You should change API Response as of our uses.
ex: `page: {
...camelizedData.page
 {number + 1 }`